docs/Examples/README.md

```diff
@@ -0,0 +1,9 @@
+Examples
+========
+
+* [Basic usage](Basic.md)
```

Comment:
I feel like we should add here basic usage at least for RxSwift, without any endpoints, just to show how it works. Below we have **SubclassingProvider** & **RxSwiftErrorHandling** with kinda more advanced usage, but we didn't really show the basics of. ReactiveCocoa would be awesome as well.